templates/compose/docker-compose.yml.erb

```diff
@@ -18,6 +18,7 @@ services:
     image: <%= scope['nexus::image'] %>:<%= scope['nexus::version'] %>
     expose:
       - 8081
+      - <%= scope['nexus::smart_proxy_port'] %>
```

Comment:
If I understand well the documentation, nexus needs to expose the port directly and not pass through the reverse proxy so this should be declared in a ``ports`` section 
To keep it simple, we can declare the external port with the same value as the internal port
- <%= scope['nexus::smart_proxy_port'] %>:<%= scope['nexus::smart_proxy_port'] %>